### ISSUE TITLE
[DOC] AFNI pipeline template

### ISSUE THREAD

**Issue by bclenet** [Inria-Empenn/narps_open_pipelines#40]
### Goal

Create a code template for AFNI-based pipelines

### Why this is useful

Templates for pipelines based on SPM and FSL already exist. Providing a code template to contributors is useful in several ways:

- it speads the expected good practices and requirements for pipeline reproduction ;
- it helps contributors to work efficiently on their pipeline reproduction.

### More specifically

Using the same idea as for these two files:

- [`narps_open/pipelines/templates/template_spm.py`](./narps_open/pipelines/templates/template_spm.py)
- [`narps_open/pipelines/templates/template_fsl.py`](./narps_open/pipelines/templates/template_fsl.py)

the template should provide tips and todos on AFNI-based pipeline writing. The [template file](./narps_open/pipelines/templates/template_afni.py) already exist but is currently blank.

### Open questions

For now, the way of implementing AFNI-based pipelines inside the project is not fully decided. The template should be written once that step is complete.
